storage/aptosdb/src/db_options.rs

```diff
@@ -71,6 +71,11 @@ pub(super) fn gen_state_merkle_cfds() -> Vec<ColumnFamilyDescriptor> {
     for cf_name in cfs {
         let mut cf_opts = Options::default();
         cf_opts.set_compression_type(DBCompressionType::Lz4);
+        let mut block_opt = schemadb::BlockBasedOptions::default();
+        block_opt.set_block_size(512);
+        block_opt
+            .set_block_cache(&schemadb::Cache::new_lru_cache(1024 * 1024 * 1024 * 50).unwrap());
```

Comment:
This value might be too crazy?

Reply:
Sorry this commit was not mean to be part of this PR, will remove.